src/ansys/dpf/composites/_composite_model_impl_2023r2.py

```diff
@@ -63,7 +63,7 @@ def __init__(
         )
 
 
-class CompositeModelInterface2023R2:
+class CompositeModelImpl2023R2:
```

Comment:
It seems odd that the descrption for this class and the one for the ``CompositeModelInterface" clss (line 34) are the same.